packages/shared/src/focusable.ts

```diff
@@ -259,12 +263,31 @@ export class Focusable extends FocusVisiblePolyfillMixin(SpectrumElement) {
         }
     }
 
+    protected override async getUpdateComplete(): Promise<boolean> {
+        const complete = (await super.getUpdateComplete()) as boolean;
+        if (this._recentlyConnected) {
+            this._recentlyConnected = false;
+            // If at connect time the [autofocus] content is placed within
+            // content that needs to be "hidden" by default, it would need to wait
+            // two rAFs for animations to be triggered on that content in
+            // order for the [autofocus] to become "visisble" and have its
+            // focus() capabilities enabled.
+            //
+            // Await this with `getUpdateComplete` so that the element cannot
+            // become "ready" until `manageFocus` has occured.
+            await nextFrame();
+            await nextFrame();
```

Comment:
Good encapsulation of this requirement & documentation, but painful that we're manually counting frames.

Reply:
This definitely should become a platform API, but we're not specifically "counting" frames as much as ensuring that a full paint cycle has gone by before this happens. Cycle is one `rAF` to get to the end of _this_ fame (which is pre-paint) and one `rAF` to get to the end of the _next_ frame (which is post paint).

Maybe the abstraction shouldn't be "next frame" and should be "paintComplete" or something?

Reply:
Understood, I'm just wary of linking rafs to any kind of logic. Perhaps there aren't extreme cases here where a particular event might be skipped due to raf not firing when we expect (or not including a particular change that we expect), but I've added bugs before by relying on ${something} happening before an raf.